utils/docker/Dockerfile.leap.15

```diff
@@ -27,7 +27,8 @@ RUN zypper --non-interactive --no-gpg-checks refresh; \
            libuuid-devel libyaml-devel                                \
            make man meson nasm ninja pandoc patch python2-pip         \
            readline-devel sg3_utils which yasm                        \
-	   python-devel python3-devel valgrind-devel
+	   python-devel python3-devel valgrind-devel hwloc-devel          \
```

Comment:
Looks like you have some extra whitespace before the `\` at the end of this line.

Reply:
done